resilience4j-monitor/src/main/java/io/github/resilience4j/monitor/internal/MonitorLogger.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *
+ *  Copyright 2023 Mariusz Kopylec
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+package io.github.resilience4j.monitor.internal;
+
+import io.github.resilience4j.monitor.LogLevel;
+import io.github.resilience4j.monitor.LogMode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class MonitorLogger {
```

Comment:
Operation execution logger. It logs execution start and end, pepending on provided configuration.